### DISCUSSION TITLE
[Atlantis] Version 5.2.0 breaks with latest versions of Primevue

### DISCUSSION THREAD

**Discussion by fotisskal** [primefaces/community#1170] [PrimeVue Templates] [answered]
Until recently I was building my project with version 5.2.0 of Atlantis template and was working perfectly with primevue 3.45.0 version. Then out of curiosity I tried to update to 3.48.1 and most elements broke. Is there any plan to document compatibility between templates and primevue versions?

**Comment by msc1979** [comment]
same for "Poseidon"

**Comment by roos-robert** [comment]
We are facing the same problem but it seems like the issue has been found and will be fixed soon (or instructions on how to fix it yourself): https://github.com/primefaces/primevue/issues/5249

**Comment by iozcelik** [comment]
Same for Apollo theme.

**Reply by malacay1462** [reply]
Same for me.
Theres also another discussion: https://github.com/orgs/primefaces/discussions/1023

**Reply by roos-robert** [reply]
Thank you for lifting this, just like people were afraid of in that thread it got buried quite quickly. Good to have it bumped through this thread to hopefully make the team aware of these issues. :)

**Comment by fotisskal** [comment]
Any update on this? Can we safely upgrade primevue library to 3.49.1 ?

**Reply by roos-robert** [reply]
Seems to still be broken, no updates out for any of the themes as of now (tested Apollo 5.0.0).

**Comment by powolnymarcel** [comment]
Still no updates :( facing the same issues

**Comment by fotisskal** [comment]
In general, their response to tickets and questions has decreased dramatically the last 2 months. This keeps me quite frustrated personally, especially when I intend to buy in the future the extended license of one of their templates. I understand they are focusing on a new release, but customer satisfaction comes first.

**Comment by powolnymarcel** [comment, marked as answer]
Woohooo got an answer from the big boss ;)

_"All templates will be updated next week to the latest PrimeVue version, and with v4 it would be trivial to self update due to new theming API enhancements. 
Regards,
Cagatay Civici
PrimeTek"_
